docs/reference/inference/chat-completion-inference.asciidoc

```diff
@@ -34,9 +34,11 @@ However, if you do not plan to use the {infer} APIs to use these models or if yo
 The chat completion {infer} API enables real-time responses for chat completion tasks by delivering answers incrementally, reducing response times during computation.
 It only works with the `chat_completion` task type for `openai` and `elastic` {infer} services.
 
+
 [NOTE]
 ====
-The `chat_completion` task type is only available within the _unified API and only supports streaming.
+* The `chat_completion` task type is only available within the _unified API and only supports streaming.
+* The Chat completion {infer} API and the Stream {infer} API differ in their response structure. If you use the `openai` service or the `elastic` service, use the Chat completion {infer} API.
```

Comment:
It might also be worth mentioning that another difference is the capabilities. The chat completion api provides a much more comprehensive API (many more fields and customization through function calling support) than the streaming api via the `completion` task type.

Basically if a user wants any short of chat history type of interaction of with a service they'll definitely want to use the chat completion functionality over the `completion` (stream) API.